docs/checkers/index.md

```diff
@@ -1566,7 +1573,7 @@ class ExampleClass:
         self.inst_attr2 = True
 ```
 
-### Pycodestyle errors (E9989) [](#9989)
+### Pycodestyle errors (E9989) [](#E9989)
```

Comment:
This change is a good one, but please revert it here and submit a separate pull request for it.